src/components/Profile.js

```diff
@@ -96,6 +90,15 @@ function Profile() {
     setShowSection(MY_SCHEDULE);
   };
 
+  function logOut() {
+    signOut(auth)
+      .then(() => {
+        props.setLogin(false);
+        localStorage.clear();
```

Comment:
comment added :

// for the planned functions now, the localStorage would be better be clear out when user logout (including user data and pin image url), so that the next user wouldn't be affect at all.